components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/TransferResource.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.identity.api.server.application.management.v1.core;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.MediaType;
+
+/**
+ * TransferResource
```

Comment:
Let's improve this class comment by adding more details. And make sure the comment 
1. Start with a capital letter
2. Ends with fullstop.

Check on other places as well